actors/util/adt/array.go

```diff
@@ -134,6 +134,17 @@ func (a *Array) Get(k uint64, out cbor.Unmarshaler) (bool, error) {
 	}
 }
 
+// MustGet retrieves array element into the 'out' unmarshaler, returning an error if it 
+//  failed to get the element or if it did not find the element in the array.
+func (a *Array) MustGet(k uint64, out cbor.Unmarshaler) error {
+	if found, err := a.root.Get(a.store.Context(), k, out); err != nil {
+		return xerrors.Errorf("failed to get index %v in root %v: %w", k, a.root, err)
```

Comment:
nit: idiomatically use type specific printing directive if available. Cids have a String method so %s works for them.  You are already doing errors the right way with the w: `"failed to get index %d in root %s: %w", k, a.root, err)`

Reply:
Also:

- The boring stuff: https://pkg.go.dev/fmt#hdr-Printing
- The fun stuff: https://pkg.go.dev/fmt#Formatter, https://pkg.go.dev/fmt#GoStringer, https://pkg.go.dev/fmt#Stringer